src/main/scala/uk/gov/hmrc/cache/TimeToLive.scala

```diff
@@ -16,15 +16,13 @@
 
 package uk.gov.hmrc.cache
 
-import play.api.Play
+import com.typesafe.config.{Config, ConfigFactory}
+import scala.concurrent.duration.DurationLong
 
 trait TimeToLive {
 
-  import scala.concurrent.duration.{MINUTES, Duration}
+  private[cache] def config: Config = ConfigFactory.load()
```

Comment:
Play 2.8 no longer supports `Play.current`. config is loaded from standard classpath.